parachains/runtimes/assets/statemine/src/lib.rs

```diff
@@ -768,7 +768,11 @@ pub type UncheckedExtrinsic =
 /// Extrinsic type that has already been checked.
 pub type CheckedExtrinsic = generic::CheckedExtrinsic<AccountId, RuntimeCall, SignedExtra>;
 /// Migrations to apply on runtime upgrade.
-pub type Migrations = ();
+pub type Migrations = (
+	cumulus_pallet_parachain_system::migration::Migration<Runtime>,
+	cumulus_pallet_xcmp_queue::migration::Migration<Runtime>,
+	cumulus_pallet_dmp_queue::migration::Migration<Runtime>,
```

Comment:
We probably already have applied these migrations?

Reply:
but isn't is safer to keep it here for next time we update these chains?